lib/tizen_sdk.dart

```diff
@@ -225,6 +229,32 @@ class TizenSdk {
     );
   }
 
+  Future<RunResult> buildDotnet(
+    String workingDirectory, {
+    String? configuration,
+    String? sign,
```

Comment:
nit: should match `tz`'s option names for better readability
```suggestion
    String? buildType,
    String? signingProfile,
```

Reply:
done!